sdk/cosmosdb/cosmos/samples-dev/typescript/src/AlterQueryThroughput.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Updates a container offer to change query throughput.
+ */
+
+import path from "path";
+import * as dotenv from "dotenv";
+dotenv.config({ path: path.resolve(__dirname, "../../../sample.env") });
+
+import { finish, handleError, logStep, logSampleHeader } from "../Shared/handleError";
+import {
+  CosmosClient,
+  OfferDefinition,
+  Resource,
+  ContainerDefinition,
+  DatabaseDefinition,
+  FeedResponse
+} from "../../../dist-esm";
+const {
+  COSMOS_DATABASE: databaseId,
+  COSMOS_CONTAINER: containerId,
+  COSMOS_ENDPOINT: endpoint,
+  COSMOS_KEY: key
+} = process.env;
+
+logSampleHeader("Alter Query Throughput");
+
+// Establish a new instance of the CosmosClient to be used throughout this demo
+const client = new CosmosClient({ endpoint, key });
+
+// ensuring a database exists for us to work with
+async function run(): Promise<void> {
+  const { database } = await client.databases.createIfNotExists({ id: databaseId });
+
+  logStep(`Create container with id : ${containerId}`);
+  await database.containers.createIfNotExists({ id: containerId });
+
+  logStep("Read all offers issued");
+  const { resources: offers } = await client.offers.readAll().fetchAll();
+
+  const newRups = 700;
+  await asyncForEach(offers, async (offerDefinition: OfferDefinition) => {
+    await updateOfferForCollection(newRups, databaseId, containerId, offerDefinition);
+  });
```

Comment:
Feels more obvious to use a simple for loop here instead of a bespoke async-ish version of forEach.

```suggestion
for (const offerDefinition of offers) {
  await updateOfferForCollection(newRups, databaseId, containerId, offerDefinition);
}
```